client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from '../../components/basic-widget';
+import { getLink } from '../../lib/nav-utils';
+import ReadingWidget from '../../components/reading-widget';
+import ShareWidget from '../../components/share-widget';
+import WidgetGroup from '../../components/widget-group';
+
+class ManageNoOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	renderShareWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Congratulations! Now that your setup is complete you can spread the word' +
+					' and get those first orders rolling in. Share your store with friends,' +
+					' family, and followers now.' ) }
+				title={ translate( 'Your store is ready, the world awaits!' ) }
+
+				urlToShare={ getLink( 'https://:site', site ) }
+			/>
+		);
+	}
+
+	renderReadingWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ReadingWidget
+				className="dashboard__reading-widget"
+				site={ site }
+				text={ translate( 'You’re not alone! Get tips from seasoned merchants,' +
+					' learn best practices to keep your store ship-shape,' +
+					' and find how to boost your sales and drive traffic.' ) }
+				title={ translate( 'Recommended reading' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Recommended Reading' )` *ES Score: 22*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-dashboard-fledgling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).